reader_core/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "pokereader"
-version = "0.7.4"
+version = "0.8.4"
```

Let's clear the patch version:
```suggestion
version = "0.8.0"
```